component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceDAOImpl.java

```diff
@@ -88,4 +88,49 @@ public SpaceEntity getSpaceByPrettyName(String spacePrettyName) {
     }
   }
 
+  @Override
+  public List<SpaceEntity> getCommonSpaces(String userId, String otherUserId, int offset, int limit) {
+    if (userId == null || userId == "") {
```

Comment:
Be careful, to test string content, you should use equals : 
```suggestion
    if (userId == null || userId.equals("")) {
```
You can also do 
```suggestion
    if (userId == null || userId.isEmpty()) {
```